src/app/modules/group/components/group-join-by-code/group-join-by-code.component.ts

```diff
@@ -32,7 +32,7 @@ export class GroupJoinByCodeComponent implements OnChanges {
   ) { }
 
   ngOnChanges() {
-    this.groupExt = withCodeAdditions(this.group);
+    this.groupExt = this.group ? withCodeAdditions(this.group) : undefined;
```

Comment:
This can be changed to `this.groupExt = this.group && withCodeAdditions(this.group)` for clarity

Reply:
This is more elegant but I would not say it brings clarity. This is very cryptic. If you show ` this.group && withCodeAdditions(this.group)`, 95% of developers will wrongly guess what it returns exactly. In most languages, it will result in an error or a boolean. 
At least it is not consistent with the rest of the code as I never use this JS/TS construct. I have to think if is a good idea to use these.